### PR TITLE
readme: update doc following install changes

### PR DESCRIPTION
**Pull request by tormath1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,29 +46,27 @@ git clone git@github.com:cycloidio/youdeploy-cli.git
...
 
-Before playing with the CLI, you first need to authenticate a user into the Cycloid API:
+## Contributing
 
-```
-cy login --org my-org --email example@email.com --password my-password --api-url https://cycloid-api.local.tld
-```
-
-From there, you can now explore the various commands using the `--help` flag for each command / subcommand.
+Have a look to our [CONTRIBUTING.md](https://github.com/cycloidio/cycloid-cli/blob/develop/CONTRIBUTING.md)
```
Comment (sdurrheimer) [inline README.md:68]:
```suggestion
Have a look to our [CONTRIBUTING.md](CONTRIBUTING.md)
```

File: README.md [hunk 2]
```diff
@@ -13,29 +46,27 @@ git clone git@github.com:cycloidio/youdeploy-cli.git
...
 
-## Common actions
+## License
 
-:construction:
-<!-- This is where we could add some useful examples: create a user, etc. -->
+This project is under MIT License, see [License](https://github.com/cycloidio/cycloid-cli/blob/develop/LICENSE)
```
Comment (sdurrheimer) [inline README.md:72]:
```suggestion
This project is under MIT License, see [License](LICENSE)
```

**Review by sdurrheimer — CHANGES_REQUESTED** [review]
It not necessary to put full links, GitHub figures it out.

**Comment by tormath1** [comment]
@sdurrheimer done :)

File: README.md [hunk 3]
```diff
@@ -65,8 +65,8 @@ You can download the latest Linux binary from the [release](https://github.com/c
 
 ## Contributing
 
-Have a look to our [CONTRIBUTING.md](https://github.com/cycloidio/cycloid-cli/blob/develop/CONTRIBUTING.md)
+Have a look to our [CONTRIBUTING.md](CONTRIBUTING.md)
 
 ## License
 
-This project is under MIT License, see [License](https://github.com/cycloidio/cycloid-cli/blob/develop/LICENSE)
+This project is under MIT License, see [License](LICENSE)
```
Comment (sdurrheimer) [inline README.md:72]:
```suggestion
This project is under MIT License, see [LICENSE](LICENSE)
```

**Comment by tormath1** [comment]
@sdurrheimer done !

**Review by sdurrheimer — APPROVED** [review]

**Pull request merged**